### PR TITLE
Java sample setup

### PR DESCRIPTION
**Pull request by tumtubs**

### CODE REVIEW

File: java/src/main/java/sort/BubbleSort.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+public class BubbleSort {
+  static <T> Pair<T, T> getSwapped(Pair<T, T> pair) {
+    return new Pair<T, T>(pair.getValue(), pair.getKey());
+  }
+
+  /*
+  Classic bubble sort works by
+  - Iterating over each element
+  - For each other element, swap if out of order
+  - Optimization: if each other element is already ordered, stop iterating
```
Comment (Bertrand31) [inline java/src/main/java/sort/BubbleSort.java]: That line got me a bit confused :sweat_smile:

Comment (tumtubs) [reply java/src/main/java/sort/BubbleSort.java]: How about: "if no swaps are needed, we consider the list to be ordered and thus stop iterating" ?

Comment (Bertrand31) [reply java/src/main/java/sort/BubbleSort.java]: Sure :)

**Comment by tumtubs** [comment]
@nicholette Added the classic implementation of BubbleSort: https://github.com/nugit/LYANL/pull/3/commits/32335f1f3a353cff97ff2333740daf7ebbdcab4a

**Comment by Bertrand31** [comment]
@tumtubs After discussion with @nicholette, we figured the point of this repo is to present final algos, not to run them. So we don't have any unrelated files, any dependencies manifest, etc. Also no need to stick to file names standards I guess.

**Comment by Bertrand31** [comment]
That's also why there are no tests anywhere in the other PRs

**Pull request closed without merging**